ndk/src/native_window.rs

```diff
@@ -377,7 +378,7 @@ impl<'a> Drop for NativeWindowBufferLockGuard<'a> {
     }
 }
 
-#[cfg(feature = "api-level-26")]
+#[cfg(all(feature = "nativewindow", feature = "api-level-26"))]
```

Comment:
I'm just realizing that I'll revert this and the change for the two `enum`s below for `SurfaceTransaction` again (in an upcoming PR), as these are exposed directly from `libandroid`.